src/test/scala/uk/gov/nationalarchives/tdr/api/routes/TransferConfirmationRouteSpec.scala

```diff
@@ -0,0 +1,91 @@
+package uk.gov.nationalarchives.tdr.api.routes
+
+import java.sql.{PreparedStatement, ResultSet}
+import java.util.UUID
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.utils.TestUtils._
+import uk.gov.nationalarchives.tdr.api.utils.{TestDatabase, TestRequest}
+
+class TransferConfirmationRouteSpec extends AnyFlatSpec with Matchers with TestRequest with TestDatabase  {
+
+  private val addTransferConfirmationJsonFilePrefix: String = "json/addtransferconfirmation_"
+
+  implicit val customConfig: Configuration = Configuration.default.withDefaults
+
+  case class GraphqlMutationData(data: Option[AddTransferConfirmation], errors: List[GraphqlError] = Nil)
+  case class GraphqlQueryData(data: Option[TransferConfirmation], errors: List[GraphqlError] = Nil)
+  case class TransferConfirmation(
+                                consignmentId: Option[UUID] = None,
+                                finalOpenRecordsConfirmed: Option[Boolean] = None,
+                                legalOwnershipTransferConfirmed: Option[Boolean] = None
+                              )
+  case class AddTransferConfirmation(addTransferConfirmation: TransferConfirmation) extends TestRequest
```

Comment:
Is this `extends TestRequest` necessary? I've seen it cause problems in the past when you attach it to a case class, because it prevents ScalaTest from displaying the contents of the object because TestRequest has non-serializable properties.

Reply:
Hmm, I can remove this; I just noticed it in 4 other RouteSpec files and thought that it was required.